source/settings/settings-reference.txt

```diff
@@ -158,10 +158,10 @@ You can configure the following settings on the |compass| interface:
      - Remain logged in when using the :manual:`MONGODB-OIDC </core/security-oidc/>` 
        authentication mechanism. 
 
-   * - Use Generative AI
-     - Feature Preview
-     - Enables :ref:`natural language querying <query-natural-language>`.
-
+   * - Enable AI Features
+     - Artificial Intelligence
+     - Enables :ref:`natural language querying <query-natural-language>` and allows 
+       the use of AI features in Compass which make requests to 3rd party services.
```

Comment:
```suggestion
       the use of AI features in Compass that make requests to third-party services.
```

Reply:
This is a verbatium string from the UI, should we leave this as is?

![Screenshot 2024-04-30 at 3 05 17 PM](https://github.com/mongodb/docs-compass/assets/85948430/e40159fd-c686-4230-a38b-993b531148b3)

Reply:
We can update this on the Compass side of things if this reads better, I'm cool with either.

Reply:
I'm okay with matching the UI :)